src/project/settings/local.py.dist

```diff
@@ -31,6 +31,18 @@ TERRA_DEFAULT_MAP_SETTINGS = {
 TERRA_APPLIANCE_SETTINGS = {
     'VIEW_ROOT_PATH': 'map', # Default view
     'DEFAULT_VIEWNAME': 'test', # Default view name
-    'enabled_modules': ['User', 'DataSource', 'DataLayer'], # Enabled terra admin modules
+    'enabled_modules': ['User', 'DataSource', 'DataLayer', 'CRUD'], # Enabled terra admin modules
 }
 
+TERRA_GEOCRUD = {
```

Comment:
Can you comment by default and add line with "Uncoment  next line to enable and configure geocrud" (Also add enabled_modules after.)